cooker/cooker.py

```diff
@@ -205,6 +205,17 @@ def dir(self):
         return self.config_.build_dir('build-' + self.name_)
 
 
+    def isbuildable(self):
+        isbuildable = False
+
+        if not self.name_.startswith('.'):
+            for build in self.ancestors_ + [ self ]:
+                if build.target_ is not None:
+                    isbuildable = True
+
+        return isbuildable
```

Comment:
I have some suggestions: 

- use `buildable` instead of `isbuildable`
- there is no need for the variable to return True/False

```suggestion
    def buildable(self):
        if not self.name_.startswith('.'):
            for build in self.ancestors_ + [ self ]:
                if build.target_ is not None:
                    return True

        return False
```